algorithms/sorting/bubble_sort.py

```diff
@@ -0,0 +1,12 @@
+'''
+Bubble Sort worst time complexity occurs when every item in array is sorted - O(n^2)
```

Comment:
don't think this statement is true.

Reply:
Good catch. I didn't state that correctly. It should say "when every item is reverse sorted". Updated comments and submitted new PR.